packages/react-conformance/src/defaultTests.tsx

```diff
@@ -146,6 +146,64 @@ export const defaultTests: TestObject = {
     });
   },
 
+  /** Component file handles classname prop */
+  'component-handles-classname': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    const { Component, wrapperComponent, helperComponents = [], requiredProps, customMount = mount } = testInfo;
+
+    let el = customMount(<Component {...requiredProps} />);
+    let component = getComponent(el, helperComponents, wrapperComponent);
+
+    const defaultClassNames =
+      component
+        ?.getDOMNode()
+        ?.getAttribute('class')
+        ?.split(' ') || [];
+
+    const testClassName = 'testComponentClassName';
+    const mergedProps: Partial<{}> = {
+      ...requiredProps,
+      className: 'testComponentClassName',
+    };
+
+    el = customMount(<Component {...mergedProps} />);
+    component = getComponent(el, helperComponents, wrapperComponent);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const classNames: any = component
+      .getDOMNode()
+      .getAttribute('class')
+      ?.split(' ');
+
+    it(`handles className prop`, () => {
+      try {
+        expect(classNames.indexOf(testClassName) >= 0).toEqual(true);
```

Comment:
There's a better matcher for this: https://jestjs.io/docs/en/24.x/expect#tocontainitem
```suggestion
        expect(classNames).toContain(testClassName);
```